models/intel/handwritten-japanese-recognition-0001/description/handwritten-japanese-recognition-0001.md

```diff
@@ -2,8 +2,9 @@
 
 ## Use Case and High-Level Description
 
-This is a network for handwritten japanese text recognition scenario. It consists of VGG16-like backbone, reshape layer and a fully connected layer.
-The network is able to recognize japanese text (characters in datasets [Kondate](http://web.tuat.ac.jp/~nakagawa/database/en/kondate_about.html) and [Nakayosi](http://web.tuat.ac.jp/~nakagawa/database/en/about_nakayosi.html)).
+This is a network for handwritten Japanese text recognition scenario. It consists of a VGG16-like backbone, 
+reshape layer and a fully connected layer.
+The network is able to recognize Japanese text. For details on characters in datasets, see [Kondate](http://web.tuat.ac.jp/~nakagawa/database/en/kondate_about.html) and [Nakayosi](http://web.tuat.ac.jp/~nakagawa/database/en/about_nakayosi.html).
```

Comment:
I think you changed the meaning of this sentence. The original meaning is that the network can recognize Japanese text that consists of characters that are in these datasets.

Reply:
Right, thanks! Fixed.